src/templates/finops-hub/modules/dataFactory.bicep

```diff
@@ -4743,6 +4746,121 @@ resource pipeline_ExecuteIngestionETL 'Microsoft.DataFactory/factories/pipelines
   }
 }
 
+//------------------------------------------------------------------------------
+// import_gcp_billing_data pipeline
+//------------------------------------------------------------------------------
+resource importGCPBillingDataPipeline 'Microsoft.DataFactory/factories/pipelines@2018-06-01' = {
+  name: 'import_gcp_billing_data'
+  parent: dataFactory
+  properties: {
+    activities: [
+      {
+        name: 'convert gcp csv'
+        type: 'Copy'
+        dependsOn: [
+          {
+            activity: 'Delete Target'
+            dependencyConditions: [
+              'Completed'
+            ]
+          }
+        ]
+        policy: {
+          timeout: '0.12:00:00'
+          retry: 0
+          retryIntervalInSeconds: 30
+          secureOutput: false
+          secureInput: false
+        }
+        typeProperties: {
+          source: {
+            type: 'DelimitedTextSource'
+            storeSettings: {
+              type: 'GoogleCloudStorageReadSettings'
+              recursive: true
+              wildcardFolderPath: gcpBillingWildcardFolderPath // Parameterized value
+              wildcardFileName: '*'
+              enablePartitionDiscovery: false
+            }
+            formatSettings: {
+              type: 'DelimitedTextReadSettings'
+            }
+          }
+          sink: {
+            type: 'ParquetSink'
+            storeSettings: {
+              type: 'AzureBlobFSWriteSettings'
+            }
+            formatSettings: {
+              type: 'ParquetWriteSettings'
+            }
+          }
+          enableStaging: false
+          translator: {
+            type: 'TabularTranslator'
+            mappings: [
```

Comment:
Mappings should be stored in the config/schema folder.  We also need to be able to detect and load the correct schema version to account for future updates.